src/backend/integration/src/tests/proposal_review_commit.spec.ts

```diff
@@ -60,445 +61,462 @@ describe('Proposal Review Commit', () => {
+        const resAliceOk = extractOkResponse(resAlice);
+        expect(resAliceOk).toEqual({
+          id: expect.any(String),
+          proposal_review_commit: {
+            proposal_review_id: aliceProposalReviewId,
+            user_id: aliceId,
+            commit_sha: VALID_COMMIT_SHA_A,
+            created_at: dateToRfc3339(currentDate),
+            state: {
+              reviewed: {
+                matches_description: [true],
+                comment: ['comment alice'],
+                highlights: [],
+              },
+            },
+          },
+        });
 
-      const { proposalReviewId: bobProposalReviewId } =
-        await createProposalReview(actor, governance, bob, proposalId);
+        const { proposalReviewId: bobProposalReviewId } =
+          await createProposalReview(actor, governance, bob, proposalId);
 
-      actor.setIdentity(bob);
-      const resBob = await actor.create_proposal_review_commit({
-        proposal_review_id: bobProposalReviewId,
-        commit_sha: VALID_COMMIT_SHA_A,
-        state: {
-          reviewed: {
-            matches_description: [true],
-            comment: ['comment bob'],
-            highlights: ['highlight bob a', 'highlight bob b'],
-          },
-        },
-      });
-      const resBobOk = extractOkResponse(resBob);
-      expect(resBobOk).toEqual({
-        id: expect.any(String),
-        proposal_review_commit: {
+        actor.setIdentity(bob);
+        const resBob = await actor.create_proposal_review_commit({
           proposal_review_id: bobProposalReviewId,
-          user_id: bobId,
           commit_sha: VALID_COMMIT_SHA_A,
-          created_at: dateToRfc3339(currentDate),
           state: {
             reviewed: {
               matches_description: [true],
               comment: ['comment bob'],
               highlights: ['highlight bob a', 'highlight bob b'],
             },
           },
-        },
+        });
+        const resBobOk = extractOkResponse(resBob);
+        expect(resBobOk).toEqual({
+          id: expect.any(String),
+          proposal_review_commit: {
+            proposal_review_id: bobProposalReviewId,
+            user_id: bobId,
+            commit_sha: VALID_COMMIT_SHA_A,
+            created_at: dateToRfc3339(currentDate),
+            state: {
+              reviewed: {
+                matches_description: [true],
+                comment: ['comment bob'],
+                highlights: ['highlight bob a', 'highlight bob b'],
+              },
+            },
+          },
+        });
       });
-    });
-
-    it('should not allow a reviewer to create too many review commits', async () => {
-      const reviewer = generateRandomIdentity();
-      const reviewerId = await createReviewer(actor, reviewer);
 
-      const { proposalReviewId: proposalReviewId1 } =
-        await createProposalReview(actor, governance, reviewer);
+      it.skip('should not allow a reviewer to create too many review commits', async () => {
```

Comment:
Even running these this (and the following) test on their own is too much for the pocketic server 😆

Reply:
I believe it's the for loop that overloads the PIC server. How about we sleep for a couple hundreds of ms in each iteration?